code/controllers/subsystem/explosion.dm

```diff
@@ -153,12 +153,12 @@ SUBSYSTEM_DEF(explosions)
 		A.color = null
 		A.maptext = ""
 
-/proc/dyn_explosion(turf/epicenter, power, flash_range, adminlog = TRUE, ignorecap = TRUE, flame_range = 0, silent = FALSE, smoke = TRUE)
+/proc/dyn_explosion(turf/epicenter, power, flash_range, adminlog = TRUE, ignorecap = TRUE, flame_range = 0, silent = FALSE, smoke = TRUE, list/explosion_overrides = list(1, 1, 1, 1))
```

Comment:
explosion_overrides should be explosion_multiplier and I think should be a datum or seperate parameters.
We are kind of reaching the point where all these parameters should be wrapped into an explosion datum to be honset.

Reply:
did the explosion_multiplier part, thinking if the datumization should be atomized out or not